packages/taquito/src/contract/rpc-contract-provider.ts

```diff
@@ -1,28 +1,38 @@
 import { HttpResponseError, STATUS_CODE } from '@taquito/http-utils';
 import { BigMapKeyType, MichelsonMap, MichelsonMapKey, Schema } from '@taquito/michelson-encoder';
-import { OpKind, ScriptResponse } from '@taquito/rpc';
+import {
+  OpKind,
+  OperationContentsBallot,
```

Comment:
not sure if it's intentional but love this alphabetic order 🙌

Reply:
I credit github copilot for this 😄